`No instances of com.sun.management.internal.OperatingSystemImpl` allowed with `-Dquarkus.native.builder-image=registry-proxy.engineering.redhat.com/rh-osbs/quarkus-mandrel-23-rhel8:23.0`

Steps to reproduce:

```
cd quarkus-cxf
git reset --hard 9619ccfdd398569b7d0bcf14cdb570475ec8542f
mvnd clean install -DskipTests -Dquarkus.build.skip
cd integration-tests/ws-security-server
mvn clean verify -Pnative -Dquarkus.native.container-build -Dquarkus.native.builder-image=registry-proxy.engineering.redhat.com/rh-osbs/quarkus-mandrel-23-rhel8:23.0

...
Checking status of builder image 'registry-proxy.engineering.redhat.com/rh-osbs/quarkus-mandrel-23-rhel8:23.0'
23.0: Pulling from rh-osbs/quarkus-mandrel-23-rhel8
Digest: sha256:d1ac4b7c86f03f3b202a2f086261f60a6236dd8a08db74f00fcdd80c0287b6ec
Status: Image is up to date for registry-proxy.engineering.redhat.com/rh-osbs/quarkus-mandrel-23-rhel8:23.0
registry-proxy.engineering.redhat.com/rh-osbs/quarkus-mandrel-23-rhel8:23.0
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildStep] Running Quarkus native-image plugin on native-image 17.0.8 2023-07-18
OpenJDK Runtime Environment Mandrel-23.0.1.0-1b3 (build 17.0.8+7-LTS)
OpenJDK 64-Bit Server VM Mandrel-23.0.1.0-1b3 (build 17.0.8+7-LTS, mixed mode)
[INFO] [io.quarkus.deployment.pkg.steps.NativeImageBuildRunner] docker run --env LANG=C --rm --user 1000:1000 -v /home/ppalaga/orgs/cxf/quarkiverse-cxf/integration-tests/ws-security-server/target/quarkus-cxf-integration-test-ws-security-server-2.3.1-SNAPSHOT-native-image-source-jar:/project:z --name build-native-jEjrf registry-proxy.engineering.redhat.com/rh-osbs/quarkus-mandrel-23-rhel8:23.0 -J-Dsun.nio.ch.maxUpdateArraySize=100 -J-Dorg.ehcache.sizeof.AgentSizeOf.bypass=true -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager -J-Dlogging.initial-configurator.min-level=500 -J-Dcom.sun.xml.bind.v2.bytecode.ClassTailor.noOptimize=true -J-Dvertx.logger-delegate-factory-class-name=io.quarkus.vertx.core.runtime.VertxLogDelegateFactory -J-Dvertx.disableDnsResolver=true -J-Dio.netty.leakDetection.level=DISABLED -J-Dio.netty.allocator.maxOrder=3 -J-Duser.language=en -J-Duser.country=IE -J-Dfile.encoding=UTF-8 --features=io.quarkus.runner.Feature,io.quarkus.runtime.graal.DisableLoggingFeature -J--add-exports=java.security.jgss/sun.security.krb5=ALL-UNNAMED -J--add-opens=java.base/java.text=ALL-UNNAMED -J--add-opens=java.base/java.io=ALL-UNNAMED -J--add-opens=java.base/java.lang.invoke=ALL-UNNAMED -J--add-opens=java.base/java.util=ALL-UNNAMED -H:BuildOutputJSONFile=quarkus-cxf-integration-test-ws-security-server-2.3.1-SNAPSHOT-runner-build-output-stats.json -H:+AllowFoldMethods -J-Djava.awt.headless=true --no-fallback --link-at-build-time -H:+ReportExceptionStackTraces -H:-AddAllCharsets --enable-url-protocols=http,https -H:-UseServiceLoaderFeature -H:+StackTrace -H:AdditionalSecurityProviders=org.apache.jcp.xml.dsig.internal.dom.XMLDSigRI -J--add-exports=org.graalvm.sdk/org.graalvm.nativeimage.impl=ALL-UNNAMED --exclude-config io\.netty\.netty-codec /META-INF/native-image/io\.netty/netty-codec/generated/handlers/reflect-config\.json --exclude-config io\.netty\.netty-handler /META-INF/native-image/io\.netty/netty-handler/generated/handlers/reflect-config\.json quarkus-cxf-integration-test-ws-security-server-2.3.1-SNAPSHOT-runner -jar quarkus-cxf-integration-test-ws-security-server-2.3.1-SNAPSHOT-runner.jar
========================================================================================================================
GraalVM Native Image: Generating 'quarkus-cxf-integration-test-ws-security-server-2.3.1-SNAPSHOT-runner' (executable)...
========================================================================================================================
Warning: Could not resolve class org.apache.cxf.javascript.JavascriptServerListener for reflection configuration. Reason: java.lang.ClassNotFoundException: org.apache.cxf.javascript.JavascriptServerListener.
[1/8] Initializing...                                                                                    (6.7s @ 0.36GB)
 Java version: 17.0.8+7-LTS, vendor version: Mandrel-23.0.1.0-1b3
 Graal compiler: optimization level: 2, target machine: x86-64-v3
 C compiler: gcc (redhat, x86_64, 8.5.0)
 Garbage collector: Serial GC (max heap size: 80% of RAM)
 4 user-specific feature(s)
 - io.quarkus.runner.Feature: Auto-generated class by Quarkus from the existing extensions
 - io.quarkus.runtime.graal.DisableLoggingFeature: Disables INFO logging during the analysis phase
 - org.eclipse.angus.activation.nativeimage.AngusActivationFeature
 - org.eclipse.angus.mail.nativeimage.AngusMailFeature
08:47:38,176 INFO  [org.ehc.siz.fil.AnnotationSizeOfFilter] Using regular expression provided through VM argument org.ehcache.sizeof.filters.AnnotationSizeOfFilter.pattern for IgnoreSizeOf annotation : ^.*cache\..*IgnoreSizeOf$
[2/8] Performing analysis...  [******]                                                                  (32.3s @ 3.46GB)
  22,614 (90.71%) of 24,930 types reachable
  33,633 (63.27%) of 53,162 fields reachable
 114,752 (62.12%) of 184,727 methods reachable
   8,302 types, 1,403 fields, and 9,715 methods registered for reflection
      64 types,    70 fields, and    55 methods registered for JNI access
       4 native libraries: dl, pthread, rt, z

Error: No instances of com.sun.management.internal.OperatingSystemImpl are allowed in the image heap as this class should be initialized at image runtime. To see how this object got instantiated use --trace-object-instantiation=com.sun.management.internal.OperatingSystemImpl.
Detailed message:
Trace: Object was reached by
  trying to constant fold static field org.ehcache.shadow.org.terracotta.offheapstore.util.PhysicalMemory.OS_BEAN
    at org.ehcache.shadow.org.terracotta.offheapstore.util.PhysicalMemory.getAttribute(PhysicalMemory.java:56)
  parsing method org.ehcache.shadow.org.terracotta.offheapstore.util.PhysicalMemory.getAttribute(PhysicalMemory.java:55) reachable via the parsing context
    at static root method.(Unknown Source)


com.oracle.svm.core.util.UserError$UserException: No instances of com.sun.management.internal.OperatingSystemImpl are allowed in the image heap as this class should be initialized at image runtime. To see how this object got instantiated use --trace-object-instantiation=com.sun.management.internal.OperatingSystemImpl.
Detailed message:
Trace: Object was reached by
  trying to constant fold static field org.ehcache.shadow.org.terracotta.offheapstore.util.PhysicalMemory.OS_BEAN
    at org.ehcache.shadow.org.terracotta.offheapstore.util.PhysicalMemory.getAttribute(PhysicalMemory.java:56)
  parsing method org.ehcache.shadow.org.terracotta.offheapstore.util.PhysicalMemory.getAttribute(PhysicalMemory.java:55) reachable via the parsing context
    at static root method.(Unknown Source)


        at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.UserError.abort(UserError.java:85)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.FallbackFeature.reportAsFallback(FallbackFeature.java:248)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:798)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:582)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:539)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:408)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:612)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.start(NativeImageGeneratorRunner.java:134)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:94)
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of com.sun.management.internal.OperatingSystemImpl are allowed in the image heap as this class should be initialized at image runtime. To see how this object got instantiated use --trace-object-instantiation=com.sun.management.internal.OperatingSystemImpl.
Detailed message:
Trace: Object was reached by
  trying to constant fold static field org.ehcache.shadow.org.terracotta.offheapstore.util.PhysicalMemory.OS_BEAN
    at org.ehcache.shadow.org.terracotta.offheapstore.util.PhysicalMemory.getAttribute(PhysicalMemory.java:56)
  parsing method org.ehcache.shadow.org.terracotta.offheapstore.util.PhysicalMemory.getAttribute(PhysicalMemory.java:55) reachable via the parsing context
    at static root method.(Unknown Source)


        at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.constraints.UnsupportedFeatures.report(UnsupportedFeatures.java:126)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:793)
        ... 6 more
```

--- a/extensions/ws-security/deployment/src/main/java/io/quarkiverse/cxf/ws/security/deployment/EhcacheProcessor.java
+++ b/extensions/ws-security/deployment/src/main/java/io/quarkiverse/cxf/ws/security/deployment/EhcacheProcessor.java
@@ -116,6 +116,7 @@ public class EhcacheProcessor {
                 "org.ehcache.sizeof.impl.JvmInformation",
                 "org.ehcache.shadow.org.terracotta.utilities.io.Files",
                 "org.ehcache.shadow.org.terracotta.offheapstore.storage.portability.BooleanPortability",
+                "org.ehcache.shadow.org.terracotta.offheapstore.util.PhysicalMemory",
                 "org.ehcache.xml.XmlConfiguration",
                 "org.ehcache.xml.ResourceConfigurationParser")
                 .map(RuntimeInitializedClassBuildItem::new)

